### PR TITLE
docs: add `and`, `or` validator to configuration tree

### PR DESCRIPTION
**Pull request by shine2lay**
Context
-----
`And` and `Or` validator are not currently showing up in readTheDocs because it is not properly added to configuration tree

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -75,6 +76,7 @@ Validator List
     validators/headRef.rst
     validators/label.rst
     validators/milestone.rst
+    validators/or.rst
```
Comment (jusx) [inline docs/configuration.rst:79]: Thoughts on adding these under advanced operators?

Comment (shine2lay) [reply docs/configuration.rst:79]: you mean a page called `advanced operators?(validator?)` and put these in them?

Comment (jusx) [reply docs/configuration.rst:79]: Yes. These are operators. So on the side bar you'd have `Validators`, `Operators` and `Actions`

Comment (shine2lay) [reply docs/configuration.rst:79]: addressed

Comment (jusx) [reply docs/configuration.rst:79]: Since you've added the Operators section and listed the operators there, should it still be on the validator list?

Comment (shine2lay) [reply docs/configuration.rst]: you are right, it was an oversight, fixed

**Review by jusx — APPROVED** [review]

**Pull request merged**